releases/release_notes_09_2022.md

```diff
@@ -0,0 +1,282 @@
+ 
+FIELDS:
+   apiVersion	<string>
+     APIVersion defines the versioned schema of this representation of an
+     object. Servers should convert recognized schemas to the latest internal
+     value, and may reject unrecognized values. More info:
+     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+ 
+   kind	<string>
+     Kind is a string value representing the REST resource this object
+     represents. Servers may infer this from the endpoint the client submits
+     requests to. Cannot be updated. In CamelCase. More info:
+     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+ 
+   metadata	<Object>
+     Standard object's metadata. More info:
+     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+ 
+   spec	<Object>
+     CcRuntimeSpec defines the desired state of CcRuntime
+ 
+   status	<Object>
+     CcRuntimeStatus defines the observed state of CcRuntime
+ ```
+
+
+The complete CRD can be seen by running the following command:
+
+```
+kubectl explain --recursive=true ccruntimes.confidentialcontainers.org
+```
+
+You can also see the details of the CcRuntime CRD in the following .go file: https://github.com/confidential-containers/operator/blob/main/api/v1beta1/ccruntime_types.go#L90
+
+Create the custom resource:
+
+```
+kubectl apply  -f https://raw.githubusercontent.com/confidential-containers/operator/main/config/samples/ccruntime.yaml
+```
+
+Check that the ccruntime was created successfully: 
+```
+kubectl get ccruntimes
+```
+Output: 
+``` 
+NAME               AGE
+ccruntime-sample   5s
+```
+
+Use the following command to observe the details of the CR yaml::
+```
+kubectl get ccruntimes ccruntime-sample -o yaml | less
+```
+
+Note that we are using  runtimeName: kataataame: kata
+
+If we were use enclave-cc for example we would observe that runtimeName: enclave-cc
+
+Once we also create the custom resource the validation will show us 2 additional pods created:
+```
+kubectl get pods -n confidential-containers-system
+```
+Output: 
+```
+NAME                                              READY   STATUS    RESTARTS   AGE
+cc-operator-controller-manager-5df7584679-kffzf   2/2     Running   0          21m
+cc-operator-daemon-install-xz697                  1/1     Running   0          6m45s
+cc-operator-pre-install-daemon-rtdls              1/1     Running   0          7m2s
+```
+
+Once the CR was created you will notice we have multiple runtime classes:
+```
+kubectl get runtimeclass
+```
+Output: 
+```
+NAME            HANDLER         AGE
+kata            kata            9m55s
+kata-clh        kata-clh        9m55s
+kata-clh-tdx    kata-clh-tdx    9m55s
+kata-qemu       kata-qemu       9m55s
+kata-qemu-tdx   kata-qemu-tdx   9m55s
+```
+
+Details on each of the runtime classes:
+
+-- kata - standard kata runtime using the QEMU hypervisor including all COCO building blocks for a non CC HW
```

Comment:
These aren't rendering as list items due to the double dash, but also we should quote the names to make them render verbatim:

```suggestion
- `kata` - standard Kata Containers runtime using the QEMU hypervisor including all COCO building blocks for a non CC HW
```

It might be worth explaining what this is for - how is it different to vanilla Kata Containers?